<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>598</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите номер удаляемого пользователя:     </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IndexInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>393</y>
     <width>800</width>
     <height>207</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>№</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Имя</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Отчество</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Телефон</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Почта</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Search">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>240</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Найти Пользователя</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>240</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить Пользователя</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>240</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить Пользователя</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>598</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите фамилию искомого пользователя:     </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="SurnameInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Return">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>350</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
